### DISCUSSION TITLE
Downloading dataset of cement industry.

### DISCUSSION THREAD

**Discussion by RogerKwak** [iiasa/message_ix#872] [Q&A]
Dear researchers,

Hello, I am a student conducting energy model-related research at Korea University in Korea.

We are making a bottom-up model of the cement industry among industrial sectors, and we are looking for a cement database.

I'm looking for cement data used in MESSAGEix-materials, and it seems like the method is to implement the model through GAMS or Python and extract excel dataset.

For example, Excel files like below. (https://docs.messageix.org/projects/models/en/latest/material/index.html)
CEMENT.BvR2010.xlsx
Global steel cement MESSAGE.xlsx

I'm wandering around with no clue right now.

Is there a place to publicly download the Korean Cement Dataset without implementing a model? How can I access it?

**Comment by glatterf42** [comment]
Hi @RogerKwak, thanks for raising this question! I have asked my colleagues working on the materials module to please take a look because they are likely better suited to answer your question. In the meantime, could you please clarify what you are interested in? Would you like to receive the historical industry data, the most current data related to cement, or some projections of future demand?

**Reply by glatterf42** [reply]
Thanks to @yiyi1991, I can forward you this suggestion: 
If you're interested in the projection or future data of cement, [this dataset](https://zenodo.org/records/10807824) contains MESSAGEix runs with the material module for the current policy and 1.5-degree policy scenarios. It has part of the results downscaled to Korea, though just `Final energy|Industry` (please also see the corresponding [scenario explorer](https://data.ene.iiasa.ac.at/ngfs/#/workspaces) with open access).

**Reply by RogerKwak** [reply]
Thanks for the reply @glatterf42 

We are making a bottom-up model for the combination of carbon reduction technologies in the cement industry.

Material flow and carbon reduction technology between cement processes used in the MESSAGEix model,
 cement demand, production, CAPEX, OPEX, lifetime, fuel consumption, learning rate by technology.

And also looking for basic economic and social datasets used in the model.


I was able to find several cement DB on this github site, but even after downloading all the .xlsx files, they do not open.(CEMENT.BvR2010, STEEL_database_2012, China_steel_cement_MESSAGE, Global_steel_cement_MESSAGE)
https://github.com/iiasa/message-ix-models/tree/main/message_ix_models/data/material/steel_cement

There seems to be an error opening the file.
